metaflow/plugins/kfp/s3_sensor_decorator.py

```diff
@@ -0,0 +1,109 @@
+from metaflow.decorators import FlowDecorator
+from metaflow.exception import MetaflowException
+
+from types import FunctionType
+from typing import Tuple
+
+"""
+Within identity_formatter, which is passed in as the func parameter,
+customers have access to all variables in flow_parameters_json (which 
+include all parameters passed in through the flow at compile and run time by
+KFP) as well as a number of environment variables, including:
```

Comment:
I wouldn't explicitly list out these env vars in the code as they are likely to change. Ideally instead we'd just put a reference to the code where these are defined.

Reply:
True, and these are Zillow deployment of MF on KFP specific.  They don't reside here, and will change often.